app/views/help/new_features.md

```diff
@@ -0,0 +1,8 @@
+<!-- When updating this file, update date to flag users of new content -->
```

Comment:
Ah hah, I realized that. Are we parsing the ERB within the markdown, or is that just what you happened to use for help text?